docs/devel/cherry-picks.md

```diff
@@ -64,6 +64,13 @@ hack/cherry_pick_pull.sh upstream/release-3.14 98765
 This will walk you through the steps to propose an automated cherry pick of pull
  #98765 for remote branch `upstream/release-3.14`.
 
+#### Labeling
```

Comment:
should probably mention in the "individual cherrypicks" section that you need to add the `cherrypick-candidate` label. It's in the "batching" section, but it's not clear that it's also necessary for individual cherrypicks post x.x.0.